internal/users/localgroups/tests/tests.go

```diff
@@ -106,3 +108,49 @@ func Mockgpasswd(_ *testing.T) {
 		os.Exit(1)
 	}
 }
+
+// SetupGPasswdMock setup the gpasswd mock and return the path to the file where the commands will be written.
+//
+// Tests that require this can not be run in parallel.
+func SetupGPasswdMock(t *testing.T, localGroupsFile string) string {
```

Comment:
I thought this function was called from multiple packages and it seems it’s only called from the `users_test` package?

Reply:
In other places, we inline the code. This isn't the PR to change the other packages' tests to use this function, I think. I can update the other places after we get the PR merged.

Reply:
sounds good, let’s just not forget about it!

Reply:
I won't!